addOns/graphql/src/main/javahelp/org/zaproxy/addon/graphql/resources/help/contents/graphql.html

```diff
@@ -56,6 +56,13 @@ <h3> Actions </h3>
 
 The definitions will be imported synchronously and any warnings will be returned.
 
+<h3>GraphiQL IDE</h3>
+A customized version of the <a href="https://github.com/graphql/graphiql">GraphiQL IDE</a> can be accessed via the API.
+
+The GraphiQL IDE is made available at <code>/graphiql</code> as well as <code>OTHER/graphql/other/graphiql</code> by the
+add-on. Note that the IDE is served like any other endpoint, and a valid API key must be specified in order to access
+it.
```

Comment:
Add an example?

`<code>http://zap/graphiql?api-key=secretkey1234</code>`
(Sorry I don't remember the param name off the top of my head.)